### DISCUSSION TITLE
Nueve [MX]

### DISCUSSION THREAD

**Discussion by Carlinhos027** [iptv-org/community#387] [Show and tell]
### Channel ID

Nueve.mx

### Stream URL

https://linear-352.frequency.stream/352/hls/master/playlist.m3u8

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by BellezaEmporium** [comment]
We should create a "us_frequency.m3u8" file I think.

**Comment by freearhey** [comment]
https://linear-352.frequency.stream/352/hls/master/playlist.m3u8 (offline)
<img width="464" alt="image" src="https://user-images.githubusercontent.com/7253922/232256664-e08b1edc-467f-4113-b939-cdc9ae3681f3.png">
